cloudformation/tests/test_cluster_custom_resource.py

```diff
@@ -0,0 +1,245 @@
+"""Test cases for Cluster CloudFormation Custom Resource."""
```

Comment:
these local tests are extremely useful but need to be executed manually. I'd suggest to also include an end to end test (even as part of a separate review) so that the same can be periodically executed to validate both the develop and the released artifacts. 
For example if a new functionality is added and we forget to update the IAM policies how do we make sure to catch such errors? Or any breaking change to a dependent resource will be hard to catch (e.g. Python 3.9 reaches EOL and is dropped by Lambda, a change in the crhelper breaks the integration, the policy substack renames one of its outputs or adds a parameter)

Reply:
Agree -- this is being tracked in our backlog and should be addressed.